report_pre_thesis/src/sections/methodology.tex

```diff
@@ -1,8 +1,191 @@
+	\label{fig:ica_data}
+\end{figure}
+
+However, we do not weigh by the inverse of the instrument response function (IRF) as mentioned in \citeauthor{cleggRecalibrationMarsScience2017} before normalizing.
+The reason for this is that the practical implications of weighing by the inverse of the IRF has been subject to criticism by some of the original authors since it effectively reintroduces the noise that was removed by applying the IRF in the first place.
+As previously mentioned, the IRF is applied specifically to remove noise from the data during the processing of the EDR to CCS data.
+Therefore, weighing by the inverse of the IRF seems counterproductive based on this information.
+
+In addition, because \citeauthor{cleggRecalibrationMarsScience2017} does not mention how each of the five datasets are used for each target, we have decided to only use the first dataset for each target.
+This likely does not produce as accurate results as using all five datasets would, since we do not get a full representation of the sample that was shot at by the laser, instead only getting a partial representation from a single location.
+Nevertheless, to avoid deviating significantly from \citeauthor{cleggRecalibrationMarsScience2017}'s methodology or making unfounded assumptions about their data processing, we have opted for this more conservative approach.
+Our goal is to test each of the components of the pipeline individually rather than trying to reproduce the results to absolute perfection.
+
+Similarly, in our recreation of the pipeline, we have not incorporated outlier detection in the ICA part of the pipeline primarily because \citeauthor{cleggRecalibrationMarsScience2017} does not describe their implementation sufficiently enough for precise replication.
+They mention the use of Median Absolute Deviation for this purpose, but the specifics of their approach remain unclear.
+Our decision to omit outlier detection in the ICA phase of our pipeline is also influenced by a preference for retaining a more comprehensive dataset, despite the potential inclusion of outliers.
+Further supporting this decision is input from one of the original authors involved in the PLS1-SM aspect of the project.
+This author highlighted the extensive efforts invested in developing the ChemCam calibration dataset, suggesting a low probability of significant outlier presence.
+Given this context, introducing outlier detection for the ICA phase based on mere assumptions would be counterproductive, potentially compromising the integrity of our analysis.
+
+After examining the results in section \ref{sec:results}, we will discuss the implications of these design choices in section \ref{sec:discussion}.
+
+\subsection{Joint Approximate Diagonalization of Eigenmatrices (JADE) and Regression}
+After preprocessing the data, we are ready to perform ICA.
+This is done using the JADE algorithm, which is the same algorithm used by \citeauthor{cleggRecalibrationMarsScience2017}.
```

Comment:
We could merge the first and second paragraphs here to basically say that we use the same algorithm and hyperparameters